Funcky.DiscriminatedUnion.SourceGeneration/Funcky.DiscriminatedUnion.SourceGeneration.csproj

```
@@ -9,7 +9,7 @@
         <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
     </PropertyGroup>
     <PropertyGroup Label="NuGet Metadata">
-        <Version>1.2.0</Version>
+        <Version>1.3.0</Version>
```

Only a bug fix; no reason to bump the minor version :)
```suggestion
        <Version>1.2.1</Version>
```